src/components/component.ts

```diff
@@ -339,9 +339,10 @@ module Plottable {
       // They don't need the current URL in the clip path reference.
       var prefix = /MSIE [5-9]/.test(navigator.userAgent) ? "" : document.location.href;
       prefix = prefix.split("#")[0]; // To fix cases where an anchor tag was used
-      this._element.attr("clip-path", "url(\"" + prefix + "#clipPath" + this.getID() + "\")");
+      var clipPathId = Utils.DOM.getUniqueClipPathId();
+      this._element.attr("clip-path", "url(\"" + prefix + "#clipPath" + clipPathId + "\")");
```

Comment:
Now that I think about it, our `clipPath` IDs should include "plottable" in the name somewhere to avoid collisions.